### ISSUE TITLE
bug: Invalid schema can result in TypeError exception

### ISSUE THREAD

**Issue by ogenstad** [opsmill/infrahub#3538]
### Component

infrahubctl CLI

### Current Behavior

When loading an invalid schema with infrahubctl there's a chance that it will raise a TypeError instead of displaying an actionable error message.

```yaml
---
version: '1.0'
nodes:
  - name: Instance
    namespace: Cloud
    attributes:
      - name: name
        kind: Text
      - name: status
        kind: Dropdown
```

### Expected Behavior

A clear user friendly error should be returned.

### Steps to Reproduce

1. Create a file with this schema:

```yaml
---
version: '1.0'
nodes:
  - name: Instance
    namespace: Cloud
    attributes:
      - name: name
        kind: Text
      - name: status
        kind: Dropdown
```

2. Try to load it with `infrahubctl schema load cloud_example.yml`

```python
❯ infrahubctl schema load ../models/cloud.yml
Unable to load the schema:
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/patrick/Code/opsmill/infrahub/python_sdk/infrahub_sdk/async_typer.py:18 in runner         │
│                                                                                                  │
│   15 │   │   │                                                                                   │
│   16 │   │   │   @wraps(func)                                                                    │
│   17 │   │   │   def runner(*args: Any, **kwargs: Any) -> Any:                                   │
│ ❱ 18 │   │   │   │   return asyncio.run(func(*args, **kwargs))                                   │
│   19 │   │   │                                                                                   │
│   20 │   │   │   decorator(runner)                                                               │
│   21 │   │   else:                                                                               │
│                                                                                                  │
│ /Users/patrick/.nix-profile/lib/python3.12/asyncio/runners.py:194 in run                         │
│                                                                                                  │
│   191 │   │   │   "asyncio.run() cannot be called from a running event loop")                    │
│   192 │                                                                                          │
│   193 │   with Runner(debug=debug, loop_factory=loop_factory) as runner:                         │
│ ❱ 194 │   │   return runner.run(main)                                                            │
│   195                                                                                            │
│   196                                                                                            │
│   197 def _cancel_all_tasks(loop):                                                               │
│                                                                                                  │
│ /Users/patrick/.nix-profile/lib/python3.12/asyncio/runners.py:118 in run                         │
│                                                                                                  │
│   115 │   │                                                                                      │
│   116 │   │   self._interrupt_count = 0                                                          │
│   117 │   │   try:                                                                               │
│ ❱ 118 │   │   │   return self._loop.run_until_complete(task)                                     │
│   119 │   │   except exceptions.CancelledError:                                                  │
│   120 │   │   │   if self._interrupt_count > 0:                                                  │
│   121 │   │   │   │   uncancel = getattr(task, "uncancel", None)                                 │
│                                                                                                  │
│ /Users/patrick/.nix-profile/lib/python3.12/asyncio/base_events.py:685 in run_until_complete      │
│                                                                                                  │
│    682 │   │   if not future.done():                                                             │
│    683 │   │   │   raise RuntimeError('Event loop stopped before Future completed.')             │
│    684 │   │                                                                                     │
│ ❱  685 │   │   return future.result()                                                            │
│    686 │                                                                                         │
│    687 │   def stop(self):                                                                       │
│    688 │   │   """Stop running the event loop.                                                   │
│                                                                                                  │
│ /Users/patrick/Code/opsmill/infrahub/python_sdk/infrahub_sdk/ctl/schema.py:164 in load           │
│                                                                                                  │
│   161 │   loading_time = time.time() - start_time                                                │
│   162 │                                                                                          │
│   163 │   if response.errors:                                                                    │
│ ❱ 164 │   │   display_schema_load_errors(console=console, response=response.errors, schemas_da   │
│   165 │   │   raise typer.Exit(1)                                                                │
│   166 │                                                                                          │
│   167 │   if response.schema_updated:                                                            │
│                                                                                                  │
│ /Users/patrick/Code/opsmill/infrahub/python_sdk/infrahub_sdk/ctl/schema.py:112 in                │
│ display_schema_load_errors                                                                       │
│                                                                                                  │
│   109 │   │   │   continue                                                                       │
│   110 │   │                                                                                      │
│   111 │   │   input_label = loc_path[-1]                                                         │
│ ❱ 112 │   │   element_label = loc_path[-3][0:-1].title()                                         │
│   113 │   │                                                                                      │
│   114 │   │   input_str = error.get("input", None)                                               │
│   115 │   │   error_message = f"{element_label} {input_label}: {input_str} | {error['msg']} ({   │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: 'int' object is not subscriptable
```

**Comment by FragmentedPacket** [comment]
Please assign me for this one.

**Comment by ogenstad** [comment]
Fixed in #3558
